Orange/preprocess/preprocess.py

```diff
@@ -339,8 +342,9 @@ class Randomize(Preprocess):
                      "RandomizeMetas")
     (RandomizeClasses, RandomizeAttributes, RandomizeMetas) = RandTypes
 
-    def __init__(self, rand_type=RandomizeClasses):
+    def __init__(self, rand_type=RandomizeClasses, rand_seed=None):
         self.rand_type = rand_type
+        np.random.seed(rand_seed)
```

Comment:
When `rand_seed` is provided, it should be used for _every_ call. Here you set the seed in the constructor so only the first call will have the expected output, but subsequent calls will depend on previous invocations.

So `init` should store the seed and it should be set in `randomize` before calls to random.shuffle.

As this is an important issue that was almost overlooked we might want to add a test for this as well. E.g. construct a randomizer and use it on A and B. Then use it on C and B. The randomization of B should be the same both times.